File: week-04/day-01/petrol_station.py
class Station(object):
    gas_amount = 10000
    def refill(self):
        x = car.capacity
        Station.gas_amount -= x
        car.gas_amountc += x




class Car(object):

    def __init__(self, gas_amountc = 0, capacity = 100):
        self.gas_amountc = gas_amountc
        self.capacity = capacity

station = Station()
car = Car()
station.refill()
print(Station.gas_amount)
print(car.gas_amountc)

File: week-08/day-01/petrol-station.js
'use strict';
// Create and object called car
//  - It should store its petrol level called petrolLevel
//  - It should store its petrol capacity called petrolCapacity
//  - It should have a refill(amount) method, that increments the petrol level,
//    than returns how much petrol it consumed from the given amount
//  - Initialize the petrol level to zero and the capacity to 50 
//
// Create an object called station
//  - It should store petrol amount called petrolStorage
//  - It should have a provide(car) method that calls the refill method of the car
//    with the stored petrol amount as a parameter, then decrement the used petrol
//  - Initialize the petrol amount to 3000

let car = {
    petrolLevel: 0,
    petrolCapacity: 50,
    refill: function (amount){
      var fillup = this.petrolCapacity - this.petrolLevel;
      console.log(fillup);
      this.petrolLevel = this.petrolCapacity;
      return fillup
    }
}
let station = {
  petrolStorage: 3000,
  provide: function(amount){
    this.petrolStorage -= car.refill();
  }

}
station.provide();
console.log(car);
console.log(station);